htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -2755,7 +2755,6 @@ public function closeFiscalPeriod($fiscal_period_id, $new_fiscal_period_id, $sep
 		$error = 0;
 		$this->db->begin();
 
-		$fiscal_period->statut = Fiscalyear::STATUS_CLOSED;
```

Comment:
This property may ve used by external code. We can remove an assignement that may break compatbility only 2 versions after we announce that property must be replaced in externzl code. And to announce this, we must first have all properties double assigned everywhere in the code so externzl development still work and user know he has 2 version to upfzte its code. So it is for the moment too early to remove such lines in method, when the assignement can be used in externzl code.